spec.md

```diff
@@ -498,10 +531,6 @@ message VolumeInfo {
   // NFS share). If set, it MUST be non-zero.
   uint64 capacity_bytes = 1;
 
```

Comment:
Thanks for the clarification guys. I can't think of a good reason the CO might care other than for binding, which doesn't matter in the dynamic provisioning case. So further objection.